### ISSUE TITLE
Can't re-add Lincoln Vehicle

### ISSUE THREAD

**Issue by sdholden28** [itchannel/fordpass-ha#344]
Prior to HA 2023.10.3 and FordPass 1.57 I had a 2022 F150 and a 2018 Lincoln Navigator loaded and working correctly. When I updated HA to 2023.10.3, both devices failed to load. I updated FordPass to 1.57 and restarted. No change. I removed the F150 device and set it back up without issue. It is now working correctly. I removed the Lincoln device, but I am unable to set it back up. After inputting the username and password (I have verified it working in the smartphone app), I receive "Unexpected Error" and I'm unable to continue any further. The log error is pasted below. Any ideas?

```
2023-10-13 09:19:47.472 ERROR (MainThread) [custom_components.fordpass.config_flow] Unexpected exception
Traceback (most recent call last):
File "/config/custom_components/fordpass/config_flow.py", line 85, in async_step_user
self.vehicles = info["userVehicles"]["vehicle details"]
~~~~^^^^^^^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
```

**Comment by itchannel** [comment]
So I believe the error is that for lincoln models the endpoint I use to get Vehicle details isn't valid. I'd love to work with one of you to get what URL is used to get lincoln vehicles. Basically need to use Burp or another proxy to see the request from mobile to API so I can get the correct endpoint. 

In the meantime I'll modify the initial config to allow an "advanced" option which will be providing the VIN like originally instead of doing a vehicle lookup. This will take me some time though so maybe later this weekend.

**Comment by sdholden28** [comment]
I'd love to help! I haven't used Burp before, but it looks fairly easy to setup. I bet I can get it done and capture some traffic from the Lincoln Way app on my android. It would likely be Monday or Tuesday before I can commit some time to that, but I'll plan on it.

**Comment by itchannel** [comment]
> I'd love to help! I haven't used Burp before, but it looks fairly easy to setup. I bet I can get it done and capture some traffic from the Lincoln Way app on my android. It would likely be Monday or Tuesday before I can commit some time to that, but I'll plan on it.

If you can that would be really helpful. It's only really the vehicle list I need.

**Comment by gnosis3825** [comment]
To add to the info here. I think you might be on the right path that it's not an "issue" with the app or HA, but I'm not sure it's just getting the vehicle list. I think the Lincoln and Ford data may no longer be "linked" on their side.

Here's my thinking. In HA I had a Lincoln working through the FordPass integeration, but it stopped working a week or so ago. I have found that the vehicle is not shown in Ford when I log in on the web. I get an error when I try to add it. The error provides no description. If I log into Lincoln on the web, I can see the vehicle. Because of the generic error on the "add vehicle" attempt in Ford, it's hard to tell if this "split" is deliberate or not on their side.

**Comment by sdholden28** [comment]
Proxying traffic through Burp requires a rooted Android device these days, which I can't do with my Samsung S22. I'll have to setup a virtual device through Genymotion or something similar, so it will take a bit. I have managed to grab the initial login endpoint. Not sure if it is helpful or not. Will let you know when I have more.

This is the target address when clicking "sign in" from the Lincoln Way app.

https://sso.ci.lincoln.com/v1.0/endpoint/default/authorize?redirect_uri=lincolnapp%3A%2F%2Fuserauthorized&response_type=code&scope=openid&max_age=3600&login_hint=eyJyZWFsbSI6ICJjbG91ZElkZW50aXR5UmVhbG0ifQ%3D%3D&code_challenge=uMHeQhmiNryOKdoujKmIEn89UHe0krr0KeRu75x6MGs&code_challenge_method=S256&client_id=9fb503e0-715b-47e8-adfd-ad4b7770f73b#appID=45133B88-0671-4AAF-B8D1-99E684ED4E45&propertySpecificContentKey=mobile_lincolnway&lang=en_us

**Comment by itchannel** [comment]
I need a volunteer to test some changes I've made. Basically if it can't magically find your car it will prompt to enter the vin manually. Which is what worked for Lincoln vehicles before. 

If someone could test that would be great. Need to use the below branch:

https://github.com/itchannel/fordpass-ha/tree/1.59

I'm not going to release a Beta unless needed as I don't want other downloading this release until I've tested it's working.

**Comment by itchannel** [comment]
Ok thanks for the feedback, looks like that's a config flow error. I'll have a look at why it's generating it later tonight.

**Comment by itchannel** [comment]
@sdholden28 Please try now. I accidentally uploaded the wrong data in config_flow.py

**Comment by rml1111** [comment]
I am having the same issue as @sdholden28. I have both Ford and Lincoln vehicles, but after 1.57 both failed to load. I was able to add back the Ford vehicle, but not the Lincoln. @itchannel I updated to the branch linked above. After entering my credentials, I am prompted for a VIN, however, I am receiving a message that "VIN not found for given account". Cute

As an aside, thanks so much for your work on this amazing plug in.

**Comment by itchannel** [comment]
Can you check your logs please. 
Should have some errors and what was returned when it tried your VIN

**Comment by RehmanAdeel** [comment]
I have lincoln and still keep asking "Vin not found for given account"

**Comment by RehmanAdeel** [comment]
![image](https://github.com/itchannel/fordpass-ha/assets/30053731/528734ba-083e-4d5b-b913-9b053fc94ba3)

**Comment by itchannel** [comment]
@RehmanAdeel Can you enable debugging and then try adding it again. Should see some information if you search for "Status function before auto_token". Below that should be the output from trying to get the Status for that VIN from the function.

**Comment by sdholden28** [comment]
Here's logs from two attempts. I've redacted credentials and VIN, but I'm happy to send those to you privately if it would help with testing.

2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] Fetching token
2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] eyJraWQiOiJvdEZBX0pzWUIxLUN4SjN2aEZYNjQ4SkRQN20tTGZaOEx2NzBVbFhLNkgwIiwiYWxnIjoiUlMyNTYifQ.eyJzaWQiOiJjNjQ4MzgwYi05OWM2LTQ4MzYtYWZhNi1lOTQ4NDVlZWY5YzQiLCJ0eXAiOiJCZWFyZXIiLCJzdWIiOiIzNTdiNjgwOS1kMmJiLTQzOWUtODc4Ny02NWFmNGE0MmE5OGUiLCJpc3MiOiJodHRwczovL2FjY291bnRzLmF1dG9ub21pYy5haS9hdXRoL3JlYWxtcy9pYW0iLCJlbWFpbCI6IiIsInByZWZlcnJlZF91c2VybmFtZSI6ImZvcmRwYXNzLzA4OGQ5ZjIwLWVkOTEtNDIwOC1iOWY5LWU0NmFhZTUzYTA3NyIsImV4cCI6MTY5Nzk4ODY4NywiaWF0IjoxNjk3OTg4Mzg3LCJhY2NvdW50SWQiOiI3NDVhMmE0MC0zMzI3LTQ5NDMtYmRmOC1mNzFkOWIzODlkOGIiLCJhdWQiOiJmb3JkcGFzcy1wcm9kIiwiYXpwIjoiZm9yZHBhc3MtcHJvZCIsImRvY19hY2Nlc3MiOiJmb3JkIiwidG9rZW4tc291cmNlIjoiYWNjb3VudC1zZXJ2aWNlIiwiYXV0aF9wcm92aWRlciI6ImZvcmRwYXNzIn0.PXV5tqsau9jY46mrhMjykWGkuK7VEYjewhUlsPxZm_e_f_1G1F-whemrQa85c_z0WTYoG1NW4l9KmjHnCuxUc3_6VsJRiV4gnSW-UWgJ8VqJJArshT3ONZ5CBFUmyQ5olInlgF_p4Qd2FfDdR86bEBqqqJ-I45I2e2RKwXBHqkTOfZV2MjW7ievvwiRITpzOSgjlcNCDvnzQIAl9RUUmb4dFqCo_EURi5x-5xnMf5_jOkNZPi6U5btMW-Lt48NH3Ug_GwDX5KOpQ_tv1GGrlGIIQjvMHLcayVDUs14GJFsW_cujtc1qMRGUGlqOL0GiVee7Le55hJ4JvwGTAs8kuAA
2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] 1697988687.774225
2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] Token is valid, continuing
2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] Getting Vehicles
2023-10-22 10:26:28.305 DEBUG (MainThread) [custom_components.fordpass.config_flow] NO VEHICLES FOUND
2023-10-22 10:26:28.305 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': 'email@gmail.com', 'password': 'PASSWORDHERE', 'region': 'North America & Canada'}
2023-10-22 10:26:48.149 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': 'email@gmail.com', 'password': 'PASSWORDHERE', 'region': 'North America & Canada'}
2023-10-22 10:26:48.149 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'vin': 'VINNUMBERHERE'}
2023-10-22 10:26:48.151 DEBUG (MainThread) [custom_components.fordpass.fordpass_new] /config/custom_components/fordpass/email@gmail.com_fordpass_token.txt
2023-10-22 10:26:48.151 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] Status function before auto_token
2023-10-22 10:26:48.152 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] None
2023-10-22 10:26:48.152 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] VINNUMBERHERE
2023-10-22 10:26:48.152 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] Trying new vehicle API endpoint
2023-10-22 10:26:48.490 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] 401
2023-10-22 10:26:48.491 DEBUG (MainThread) [custom_components.fordpass.config_flow] GOT SOMETHING BACK?
2023-10-22 10:26:48.491 DEBUG (MainThread) [custom_components.fordpass.config_flow] <Response [401]>
2023-10-22 10:26:48.491 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': 'email@gmail.com', 'password': 'PASSWORDHERE', 'region': 'North America & Canada', 'vin': 'VINNUMBERHERE'}
2023-10-22 10:26:53.038 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': 'email@gmail.com', 'password': 'PASSWORDHERE', 'region': 'North America & Canada', 'vin': 'VINNUMBERHERE'}
2023-10-22 10:26:53.039 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'vin': 'VINNUMBERHERE'}
2023-10-22 10:26:53.039 DEBUG (MainThread) [custom_components.fordpass.fordpass_new] /config/custom_components/fordpass/email@gmail.com_fordpass_token.txt
2023-10-22 10:26:53.039 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] Status function before auto_token
2023-10-22 10:26:53.039 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] None
2023-10-22 10:26:53.039 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] VINNUMBERHERE
2023-10-22 10:26:53.039 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] Trying new vehicle API endpoint
2023-10-22 10:26:53.368 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] 401
2023-10-22 10:26:53.369 DEBUG (MainThread) [custom_components.fordpass.config_flow] GOT SOMETHING BACK?
2023-10-22 10:26:53.369 DEBUG (MainThread) [custom_components.fordpass.config_flow] <Response [401]>
2023-10-22 10:26:53.369 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': 'meganleighholden@gmail.com', 'password': 'XpckSQ&5aUhj7%z%zgzh', 'region': 'North America & Canada', 'vin': 'VINNUMBERHERE'}

**Comment by gnosis3825** [comment]
Did some experimenting today. I am unable to get this working even with 1.60. Here's what I did learn though, maybe it's useful info. In Ford's website I cannot add a Lincoln VIN, just get a generic error, like "retry", but in the Lincoln website when I try to add the Ford, it recognizes the VIN as a Ford and links me back to the Ford website. The reason I think this is possibly useful is that it seems Ford can longer "see" Lincoln anymore, even though Lincoln can at least "kinda see" Ford, so if the API is "looking" at the "Ford side" maybe it can't even see the Lincoln stuff? Is a different endpoint needed for Lincoln now???

**Comment by itchannel** [comment]
I've looked at the Lincoln way app and can't see it making calls to any other endpoints. Just the same oauth as Fordpass. 

Out of interest does the older version of this integration work for you using the old api?

Might need some time with someone to go through running a proxy and having a look at what we can see.

**Comment by itchannel** [comment]
> Here's logs from two attempts. I've redacted credentials and VIN, but I'm happy to send those to you privately if it would help with testing.
> 
> 2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] Fetching token 2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] eyJraWQiOiJvdEZBX0pzWUIxLUN4SjN2aEZYNjQ4SkRQN20tTGZaOEx2NzBVbFhLNkgwIiwiYWxnIjoiUlMyNTYifQ.eyJzaWQiOiJjNjQ4MzgwYi05OWM2LTQ4MzYtYWZhNi1lOTQ4NDVlZWY5YzQiLCJ0eXAiOiJCZWFyZXIiLCJzdWIiOiIzNTdiNjgwOS1kMmJiLTQzOWUtODc4Ny02NWFmNGE0MmE5OGUiLCJpc3MiOiJodHRwczovL2FjY291bnRzLmF1dG9ub21pYy5haS9hdXRoL3JlYWxtcy9pYW0iLCJlbWFpbCI6IiIsInByZWZlcnJlZF91c2VybmFtZSI6ImZvcmRwYXNzLzA4OGQ5ZjIwLWVkOTEtNDIwOC1iOWY5LWU0NmFhZTUzYTA3NyIsImV4cCI6MTY5Nzk4ODY4NywiaWF0IjoxNjk3OTg4Mzg3LCJhY2NvdW50SWQiOiI3NDVhMmE0MC0zMzI3LTQ5NDMtYmRmOC1mNzFkOWIzODlkOGIiLCJhdWQiOiJmb3JkcGFzcy1wcm9kIiwiYXpwIjoiZm9yZHBhc3MtcHJvZCIsImRvY19hY2Nlc3MiOiJmb3JkIiwidG9rZW4tc291cmNlIjoiYWNjb3VudC1zZXJ2aWNlIiwiYXV0aF9wcm92aWRlciI6ImZvcmRwYXNzIn0.PXV5tqsau9jY46mrhMjykWGkuK7VEYjewhUlsPxZm_e_f_1G1F-whemrQa85c_z0WTYoG1NW4l9KmjHnCuxUc3_6VsJRiV4gnSW-UWgJ8VqJJArshT3ONZ5CBFUmyQ5olInlgF_p4Qd2FfDdR86bEBqqqJ-I45I2e2RKwXBHqkTOfZV2MjW7ievvwiRITpzOSgjlcNCDvnzQIAl9RUUmb4dFqCo_EURi5x-5xnMf5_jOkNZPi6U5btMW-Lt48NH3Ug_GwDX5KOpQ_tv1GGrlGIIQjvMHLcayVDUs14GJFsW_cujtc1qMRGUGlqOL0GiVee7Le55hJ4JvwGTAs8kuAA 2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] 1697988687.774225 2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] Token is valid, continuing 2023-10-22 10:26:27.775 DEBUG (SyncWorker_42) [custom_components.fordpass.fordpass_new] Getting Vehicles 2023-10-22 10:26:28.305 DEBUG (MainThread) [custom_components.fordpass.config_flow] NO VEHICLES FOUND 2023-10-22 10:26:28.305 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': '[email@gmail.com](mailto:email@gmail.com)', 'password': 'PASSWORDHERE', 'region': 'North America & Canada'} 2023-10-22 10:26:48.149 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': '[email@gmail.com](mailto:email@gmail.com)', 'password': 'PASSWORDHERE', 'region': 'North America & Canada'} 2023-10-22 10:26:48.149 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'vin': 'VINNUMBERHERE'} 2023-10-22 10:26:48.151 DEBUG (MainThread) [custom_components.fordpass.fordpass_new] /config/custom_components/fordpass/[email@gmail.com_fordpass_token.txt](mailto:email@gmail.com_fordpass_token.txt) 2023-10-22 10:26:48.151 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] Status function before auto_token 2023-10-22 10:26:48.152 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] None 2023-10-22 10:26:48.152 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] VINNUMBERHERE 2023-10-22 10:26:48.152 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] Trying new vehicle API endpoint 2023-10-22 10:26:48.490 DEBUG (SyncWorker_29) [custom_components.fordpass.fordpass_new] 401 2023-10-22 10:26:48.491 DEBUG (MainThread) [custom_components.fordpass.config_flow] GOT SOMETHING BACK? 2023-10-22 10:26:48.491 DEBUG (MainThread) [custom_components.fordpass.config_flow] <Response [401]> 2023-10-22 10:26:48.491 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': '[email@gmail.com](mailto:email@gmail.com)', 'password': 'PASSWORDHERE', 'region': 'North America & Canada', 'vin': 'VINNUMBERHERE'} 2023-10-22 10:26:53.038 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': '[email@gmail.com](mailto:email@gmail.com)', 'password': 'PASSWORDHERE', 'region': 'North America & Canada', 'vin': 'VINNUMBERHERE'} 2023-10-22 10:26:53.039 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'vin': 'VINNUMBERHERE'} 2023-10-22 10:26:53.039 DEBUG (MainThread) [custom_components.fordpass.fordpass_new] /config/custom_components/fordpass/[email@gmail.com_fordpass_token.txt](mailto:email@gmail.com_fordpass_token.txt) 2023-10-22 10:26:53.039 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] Status function before auto_token 2023-10-22 10:26:53.039 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] None 2023-10-22 10:26:53.039 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] VINNUMBERHERE 2023-10-22 10:26:53.039 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] Trying new vehicle API endpoint 2023-10-22 10:26:53.368 DEBUG (SyncWorker_21) [custom_components.fordpass.fordpass_new] 401 2023-10-22 10:26:53.369 DEBUG (MainThread) [custom_components.fordpass.config_flow] GOT SOMETHING BACK? 2023-10-22 10:26:53.369 DEBUG (MainThread) [custom_components.fordpass.config_flow] <Response [401]> 2023-10-22 10:26:53.369 DEBUG (MainThread) [custom_components.fordpass.config_flow] {'username': '[meganleighholden@gmail.com](mailto:meganleighholden@gmail.com)', 'password': 'XpckSQ&5aUhj7%z%zgzh', 'region': 'North America & Canada', 'vin': 'VINNUMBERHERE'}

So looking at those logs a auto_token was never issued. That might be a bug on my end. Do you have anything else in your logs including "Getting Auto Token"

**Comment by rml1111** [comment]
@itchannel I upgraded to 1.61 beta 3 this morning and am no longer receiving the dialog to add by VIN after adding my two Ford vehicles. I was receiving this dialog in 1.59 and 1.60, but I received a "No Vehicles Found" message when attempting to add my Lincoln by VIN. I can upload logs if you can please let me know which version to use when attempting to add the Lincoln.

As an aside, I really only want to add 1 of the 2 Ford vehicles from the Ford side, however, in order to get the VIN dialog to pop up in 1.59 and 1.60, I needed to add the 2nd Ford and disable it.

**Comment by gnosis3825** [comment]
> I've looked at the Lincoln way app and can't see it making calls to any other endpoints. Just the same oauth as Fordpass.
> 
> Out of interest does the older version of this integration work for you using the old api?
> 
> Might need some time with someone to go through running a proxy and having a look at what we can see.

Thanks for the explanation. I won't pretend to understand beyond that though. I can only answer your question as "it used to work", and I didn't do anything special. I don't *think* I had either car added in either place either, just each in their own sites. That said, if there is some way I can help, I'm happy to. I think it's amazing that you can, and are, doing this.

**Comment by sdholden28** [comment]
@itchannel I don't see any mention of the Auto Token in the logs. HOWEVER, if I look in the fordpass custom_components folder, there IS a fordpass_token.txt file that matches the account with an appropriate timestamp, so it does look like it was created. The file is populated.

**Comment by RehmanAdeel** [comment]
> @RehmanAdeel Can you enable debugging and then try adding it again. Should see some information if you search for "Status function before auto_token". Below that should be the output from trying to get the Status for that VIN from the function.
> 
> Or hit me up on Discord when you have 5 as I think I accepted a request from you.

Tell me your availability so i can share with you
also please mentioned which branch i need to try

**Comment by mcnutter1** [comment]
Same issue here, happy to help with any api call research. I have a lincoln as well and the fordpass integration used to work but now doesnt.

**Comment by RehmanAdeel** [comment]
> > @RehmanAdeel Can you enable debugging and then try adding it again. Should see some information if you search for "Status function before auto_token". Below that should be the output from trying to get the Status for that VIN from the function.
> > Or hit me up on Discord when you have 5 as I think I accepted a request from you.
> 
> Tell me your availability so i can share with you also please mentioned which branch i need to try

Please guide me how to enable  debugging on it

**Comment by rml1111** [comment]
@itchannel I updated to 1.61 and attempted to re-add my Ford and Lincoln vehicles. The Ford vehicles were successfully added, however, I am no longer receiving a prompt to enter my Lincoln VIN. Instead, after entering my credentials, I am getting a dialog box with the following message: No vehicles on account or all are configured already.

Please let me know if you would like me to include any logs and thank you!

**Comment by itchannel** [comment]
@rml1111 So you have bother Ford vehicles and Lincoln vehicles tied to your Fordpass account?

Currently the code won't show the manual VIN prompt if it finds valid vehicles which is probably why it isn't showing. For debug purposes could you create a separate account and invite your lincoln vehicle to share with that account. That might be easier to debug for now.

**Comment by gnosis3825** [comment]
Seeing your last comment, I tried that. Created a new account at Ford, added a vehicle and the Ford site recognized it as a Lincoln and only gave me the option to view it at the Lincoln site, which worked. Went back to HA to add the integration, was prompted for a VIN, but errored as "VIN not found". Just wanted to mention it.

**Comment by rml1111** [comment]
@itchannel sorry for the delay in responding. I tried adding the Lincoln vehicle from a separate FordPass/Lincoln account that only has the Lincoln associated with it. I was prompted for a VIN when attempting to add the Lincoln from the separate FordPass/Lincoln account, however received a "VIN not found" message (same as @gnosis3825). Glad to send logs, if helpful. Thanks!

**Comment by aeicherly** [comment]
I'm not sure if this has been resolved, but I own a ford and lincoln and here are the steps I took:
1.  Removed my ford vehicle from ford pass
2. Enabled the integration
3. Integration found nothing, manually added lincoln VIN
4. Lincoln vehicle successfully added
5. Added ford vehicle back in ford pass
6. Reloaded the integration 
7. success it recognized my ford

****i use the same email address and password in ford pass and lincoln way

**Comment by gnosis3825** [comment]
It is working for me. Had to remove my existing vehicle from Ford to get the “add by VIN” screen to come up (otherwise it would only want to add the existing). Once removed I was able to add the Lincoln by VIN. I did not add the Ford again because it’s no longer mine, but I’m able to see the Lincoln data now. Might be an issue with battery level showing but I’m also at zero so can’t tell if it’s that or no reading.
